daemons/fenced/pacemaker-fenced.c

```diff
@@ -1377,15 +1377,216 @@ st_peer_update_callback(enum crm_status_type type, crm_node_t * node, const void
+	N_("Advanced use only: The maximum number of times to retry the 'list' command within the timeout period"),
+        N_("Some devices do not support multiple connections."
+           " Operations may 'fail' if the device is busy with another task so Pacemaker will automatically retry the operation,      if there is time remaining."
+           " Use this option to alter the number of times Pacemaker retries 'list' actions before giving up.")
+    },
+    {
+	"pcmk_monitor_action",NULL, "string", NULL, "monitor", NULL,
+	N_("Advanced use only: An alternate command to run instead of 'monitor'"),
+        N_("Some devices do not support the standard commands or may provide additional ones.\n"
+                 "Use this to specify an alternate, device-specific, command that implements the 'monitor' action.")
+    },
+    {
+	"pcmk_monitor_timeout",NULL, "time", NULL, "60s", NULL,
+	N_("Advanced use only: Specify an alternate timeout to use for monitor actions instead of stonith-timeout"),
+        N_("Some devices need much more/less time to complete than normal.\n"
+	   "Use this to specify an alternate, device-specific, timeout for 'monitor' actions.")
+    },
+    {
+	"pcmk_monitor_retries",NULL, "integer", NULL, "2", NULL,
+	N_("Advanced use only: The maximum number of times to retry the 'monitor' command within the timeout period"),
+        N_("Some devices do not support multiple connections."
+           " Operations may 'fail' if the device is busy with another task so Pacemaker will automatically retry the operation,      if there is time remaining."
+           " Use this option to alter the number of times Pacemaker retries 'monitor' actions before giving up.")
+    },
+    {
+	"pcmk_status_action",NULL, "string", NULL, "status", NULL,
+	N_("Advanced use only: An alternate command to run instead of 'status'"),
+        N_("Some devices do not support the standard commands or may provide additional ones.\n"
+                 "Use this to specify an alternate, device-specific, command that implements the 'status' action.")
+    },
+    {
+	"pcmk_status_timeout",NULL, "time", NULL, "60s", NULL,
+	N_("Advanced use only: Specify an alternate timeout to use for status actions instead of stonith-timeout"),
+        N_("Some devices need much more/less time to complete than normal.\n"
+	   "Use this to specify an alternate, device-specific, timeout for 'status' actions.")
+    },
+    {
+	"pcmk_status_retries",NULL, "integer", NULL, "2", NULL,
+	N_("Advanced use only: The maximum number of times to retry the 'status' command within the timeout period"),
+        N_("Some devices do not support multiple connections."
+           " Operations may 'fail' if the device is busy with another task so Pacemaker will automatically retry the operation,      if there is time remaining."
+           " Use this option to alter the number of times Pacemaker retries 'status' actions before giving up.")
+    },
+};
+
+void
+crmd_metadata(void)
```

Comment:
"crmd" is the old name for the pacemaker-controld daemon; "fenced_metadata" or "fencer_metadata" would be better here